federation/pkg/federation-controller/cluster/clustercontroller.go

```diff
@@ -43,7 +43,7 @@ type ClusterController struct {
 	// clusterMonitorPeriod is the period for updating status of cluster
 	clusterMonitorPeriod time.Duration
 
-	mu              sync.RWMutex
+	sync.Mutex
```

Comment:
in general, don't inline this if you don't want external callers to have access to your lock/unlock functions